test/source/tests/page_recipe.ts

```diff
@@ -338,10 +351,13 @@ export class InboxPageRecipe extends PageRecipe {
 export class ComposePageRecipe extends PageRecipe {
 
   public static openStandalone = async (
-    t: AvaContext, browser: BrowserHandle, group: CommonBrowserGroup, options:
+    t: AvaContext, browser: BrowserHandle, group: CommonBrowserGroup | string, options:
       { appendUrl?: string, hasReplyPrompt?: boolean, skipClickPropt?: boolean, initialScript?: EvaluateFn } = {}
   ): Promise<ControllablePage> => {
-    const email = (group === 'compose') ? 'test.ci.compose%40org.flowcrypt.com' : 'flowcrypt.compatibility%40gmail.com';
+    if (['compatibility', 'compose'].includes(group)) { // Most common accounts
```

Comment:
Actually here it would probably be cleaner to do one by one..
```
if(group === compose) {
  email = 'test.ci.compose@org.flowcrypt.com';
} else if (group === compatibility) {
  email = ...
} else {
  throw 
}
```
Or, like before, you could define a dictionary somewhere `ccctEmailByGroup = {[group: string]: string}`, then use that here. Maybe even without the throw, because the types should protect you.